pkgs/by-name/kc/kchat/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchurl
+, appimageTools
+}:
+
+appimageTools.wrapType2 rec {
+  pname = "kchat";
+  version = "2.4.0";
+
+  src = fetchurl {
+    url = "https://download.storage5.infomaniak.com/${pname}/${pname}-desktop-${version}-linux-x86_64.AppImage";
+    name = "${pname}-${version}.AppImage";
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Reply:
Hello, 
Thanks for letting me know. I was not aware of this. I am updating this.

Reply:
Don't rely on `pname` in the rest of the file, indeed.